### PR TITLE
`tctl bots add` outputs ca pins

### PR DESCRIPTION
**Pull request by GavinFrazar**
This PR makes `tctl bots add` outpout CA pin(s) for the example command.

If `--ca-pin` isn't given, then `tbot start ...` prints a warning:

> WARN [AUTH]      Joining cluster without validating the identity of the Auth Server. This may open you up to a Man-In-The-Middle (MITM) attack if an attacker can gain privileged network access. To remedy this, use the CA pin value provided when join token was generated to validate the identity of the Auth Server or point to a valid Certificate via the CA Path option. auth/register.go:496

Our machine-id getting started guide also states:
> `ca-pin` is the CA Pin for your Teleport cluster, and is output by the `tctl bots add` command.

## Example of new output
```diff
$ tctl bots add db-bot --roles=db-bot
The bot token: 0db5cc02a57a7f41f7dfff0a1d290bf7
This token will expire in 59 minutes.

Optionally, if running the bot under an isolated user account, first initialize
the data directory by running the following command as root:

 > tbot init \
   --destination-dir=./tbot-user \
   --bot-user=tbot \
   --reader-user=alice

... where "tbot" is the username of the bot's UNIX user, and "alice" is the
UNIX user that will be making use of the certificates.

Then, run this as the bot user to begin continuously fetching
certificates:

 > tbot start \
   --destination-dir=./tbot-user \
   --token=0db5cc02a57a7f41f7dfff0a1d290bf7 \
   --auth-server=mars.internal:3080 \
   --join-method=token \
+   --ca-pin="sha256:e9d1d784023a4adaeabd006f5ec6eafdb658115bb572d659c9ee4cb88e35d457" \
+   --ca-pin="sha256:82742a934a2c21b1490780ad783e15573ea162172e176e0b86f7c40f73e68bf3"
```

there's two pins because I ran the command during HostCA rotation. Otherwise, there's just the one.

### CODE REVIEW

**Review by atburke — APPROVED** [review]

File: tool/tctl/common/bots_command_test.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	require.Len(t, caPins, 2)
+
+	roleYAMLPath := filepath.Join(t.TempDir(), "role.yaml")
+	require.NoError(t, os.WriteFile(roleYAMLPath, []byte(botRoleYAML), 0644))
+	_, err = runResourceCommand(t, fc, []string{"create", roleYAMLPath})
+	require.NoError(t, err)
+
+	buf, err := runBotsCommand(t, fc, []string{"add", "hal9000", "--roles", "robot"})
+	require.NoError(t, err)
+	fmt.Printf("\n---HERE---\n\n%s\n---\n\n", buf.String())
```
[the remark this answers is not in the document]
Comment (GavinFrazar) [reply tool/tctl/common/bots_command_test.go:81]: lol, thank you for catching that. I was using this while debugging

[the remark this answers is not in the document]
Comment (GavinFrazar) [reply tool/tctl/common/bots_command_test.go]: removed in 45b0343

**Review by strideynet — COMMENTED** [review]
Hm, I'm a little cautious of this. The address output by `tctl bots add` is the address of a proxy. The CA pins are not used when connecting to a Proxy, but, they certainly won't do any harm.

It feels like the real root issue here is that we attempt to connect to the proxy as if it /were/ an Auth Server because we do not provide a way to properly distinguish whether you are providing a Proxy or Auth Server address and hence we have to try both.

**Comment by GavinFrazar** [comment]
> Hm, I'm a little cautious of this. The address output by `tctl bots add` is the address of a proxy. The CA pins are not used when connecting to a Proxy, but, they certainly won't do any harm.
> 
> It feels like the real root issue here is that we attempt to connect to the proxy as if it /were/ an Auth Server because we do not provide a way to properly distinguish whether you are providing a Proxy or Auth Server address and hence we have to try both.

As long as we're trying both, ca pin should be provided. I made this change to match the docs guide

**Comment by GavinFrazar** [comment]
@strideynet should I just close this then?

**Pull request closed without merging**